tool/tsh/common/tsh.go

```diff
@@ -5281,7 +5281,9 @@ func awaitRequestResolution(ctx context.Context, clt authclient.ClientI, req typ
 					return nil, trace.BadParameter("unexpected resource type %T", event.Resource)
```

Comment:
We probably shouldn't update `reqState` here if this is for a different access request.